regolith/builders/grantreportbuilder.py

```diff
@@ -36,12 +43,23 @@ def construct_global_ctx(self):
 
     def latex(self):
         """Render latex template"""
+        rc = self.rc
         # Get Dates
-        today = str(date.today()).split("-")
-        end_year, end_month, end_day = int(today[0]), int(today[1]), int(today[2])
-        begin_year, begin_month, begin_day = int(today[0]) - 1, int(today[1]), int(today[2])
-        end_date = date(end_year, end_month, end_day)
-        begin_date = date(begin_year, begin_month, begin_day)
+        start_date = rc.start_date.split("-")
```

Comment:
this is heading us back down a road to the old days (passing dates around as strings and ints) rather than the brave new world of working with dates.   I very much doubt this is the direction we want to go in here....

Reply:
Yea I realized this too! Will work with date objects instead of the strings and ints.